odoo/cli/genpwd.py

```diff
@@ -0,0 +1,34 @@
+import argparse
+import random
+import os
+import sys
+
+from . import Command
+from odoo.tools import config
+
+
+class GenPwd(Command):
+    def __init__(self):
+        self.command_name = "genpwd"
+
+    def generate_password(self, length=16, charset="abcdefghijkmnpqrstuvwxyz23456789"):
+        password = ""
+        for i in range(length):
+            if (i % 4) == 0 and i != 0:
+                password += "-"
+            password += random.choice(charset)
+        return password
+
+    def run(self, cmdargs):
+        parser = argparse.ArgumentParser(
+            prog="%s %s" % (sys.argv[0].split(os.path.sep)[-1], self.command_name),
+            description="Generate and set the admin_passwd in config file"
```

Comment:
@d-fence Generating a random super-admin password is not a bad idea (although the db manager will be turned off in theory), but the point here is to generate the secret key for the proxy access, as needed in addons/hw_l10n_eg_eta/controllers/main.py (`proxy_access_token`)

The CLI command could be named `genproxytoken` or something like that?

```suggestion
            description="Generate and (re)set proxy access token in config file"
```